net/Socket.cpp

```diff
@@ -1302,6 +1383,33 @@ LocalServerSocket::~LocalServerSocket()
 #  define LOG_CHUNK(X)
 #endif
 
+#endif // !MOBILEAPP
+
+std::string StreamSocket::toString(WSState t) noexcept
+{
+    if( WSState::WS == t )
+    {
+        return "WS";
+    }
+    return "HTTP";
+}
+
+std::string StreamSocket::toString() const noexcept
+{
+    std::string s("StreamSocket[#");
+    s.append(std::to_string(getFD()))
```

Comment:
If we must use `append`, we better `reserve`. This code will almost certainly re-allocate the string buffer more than once, unnecessarily. Either use `ostringstream` or `string` with reasonable `reserve` up-front to avoid said re-allocations.